pages/common/vim.md

```diff
@@ -4,6 +4,8 @@
 > Pressing `i` enters edit mode. `<Esc>` goes back to normal mode, which doesn't allow regular text insertion.
 > More information: <https://www.vim.org>.
 
+**Important:** If you accidentally open Vim and have never used it before, _do not_ press anything. Type `:q!` and press enter to quit.
```

Comment:
Oops! I forgot to return to this PR. Sorry about that! Upon further investigation, it looks like this line is causing the build to fail, so we probably need to remove it.